torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -884,10 +1054,12 @@ def save_checkpoint(
                     model_state_dict
                 )
 
-            # Output file is always a .pt file with the epoch number in the name
+            # TODO: We should consider adding adapter/model config, like we do for HF.
```

Comment:
Is this todo applicable? Like what would we do with these configs for a Meta-format checkpoint anyways? (Maybe I am missing the point here)

Reply:
it is just an useful map of ckpt file to weights. I guess no use in our current pipeline, but i think it makes sense to have all ckpts behave the same way, and the difference between them is mostly the model format, and **not** the metadata, naming, files, etc.